src/main/java/se/sundsvall/comfactfacade/service/SigningService.java

```diff
@@ -69,4 +87,27 @@ public se.sundsvall.comfactfacade.api.model.Signatory getSignatory(final String
 	}
 
 
+	private void fetchPersonalNumbers(final SigningInstanceInputType request) {
+
+		processEntities(request.getSignatories());
+
+		Optional.ofNullable(request.getInitiator())
+			.ifPresent(this::processEntity);
+
+		processEntities(request.getAdditionalParties());
+	}
+
+	private void processEntities(final List<? extends comfact.PartyType> partyTypes) {
+		Optional.ofNullable(partyTypes)
+			.ifPresent(parties -> parties.forEach(this::processEntity));
+	}
+
+	private void processEntity(final comfact.PartyType partyType) {
+		Optional.ofNullable(partyType)
```

Comment:
Shouldn't this service throw error if partyId cannot be translated to person number? Will there ever by a case when the signing process can be started/completed if a legalId is not present for all parties?

Reply:
if party cant find a personal number it will return an error which will bubble up to the enduser. Sure we could catch that and throw an equal error as Party did. But this is not intended to only be used by the E-sign service so I don't see any value in doing that?